zio-http/src/main/scala/zio/http/model/headers/values/Expires.scala

```diff
@@ -0,0 +1,44 @@
+package zio.http.model.headers.values
+
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
+
+sealed trait Expires {
+  def value: String
```

Comment:
I think we should represent that as `LocalDateTime` or any other date type to make it easier to work with the header

Reply:
I did have LocalDateTime at first but it would lose the timezone info when we encode and decode.  I am assuming the clients will be using this from toExpires and fromExpires, so I changed the value to String.  Should I change it to ZonedDateTime?